src/main/java/us/icarenow/web/repository/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package us.icarenow.web.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
```

Comment:
please wait for Adi's changes. Once his changes are in main branch then update your branchand adapt to his changes